src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -15,9 +13,9 @@
  */
 @WebServlet("/item-submission")
 public class ItemSubmissionServlet extends HttpServlet {
-  private static final String TITLE_PARAMETER_KEY = "title";
-  private static final String DESCRIPTION_PARAMETER_KEY = "description";
-  private static final String IMAGE_URL_PARAMETER_KEY = "imageUrl";
+  private static final String TITLE_PARAMETER_KEY = "Title";
+  private static final String DESCRIPTION_PARAMETER_KEY = "Plot";
```

Comment:
Shouldn't these names be lowercase?

Reply:
I made the "Title" and "Plot" keys capitalized because the JSON of the omdbItem has the properties Title, and Plot. If I remove the capitalization then the request can't find those parameters. Here is an example of an omdbItem JSON: 
```
{"Title":"Shrek the Third","Year":"2007","Rated":"PG","Released":"18 May 2007","Runtime":"93 min","Genre":"Animation, Adventure, Comedy, Family, Fantasy","Director":"Chris Miller, Raman Hui(co-director)","Writer":"Jeffrey Price (screenplay by), Peter S. Seaman (screenplay by), Chris Miller (screenplay by), Aron Warner (screenplay by), Andrew Adamson (story by), William Steig (based upon the book by)","Actors":"Mike Myers, Eddie Murphy, Cameron Diaz, Antonio Banderas","Plot":"When his new father-in-law, King Harold falls ill, Shrek is looked at as the heir to the land of Far, Far Away. Not one to give up his beloved swamp, Shrek recruits his friends Donkey and Puss in Boots to install the rebellious Artie as the new king. Princess Fiona, however, rallies a band of royal girlfriends to fend off a coup d'etat by the jilted Prince Charming.","Language":"English","Country":"USA","Awards":"Nominated for 1 BAFTA Film Award. Another 5 wins & 16 nominations.","Poster":"https://m.media-amazon.com/images/M/MV5BOTgyMjc3ODk2MV5BMl5BanBnXkFtZTcwMjY0MjEzMw@@._V1_SX300.jpg","Ratings":[{"Source":"Internet Movie Database","Value":"6.1/10"},{"Source":"Rotten Tomatoes","Value":"41%"},{"Source":"Metacritic","Value":"58/100"}],"Metascore":"58","imdbRating":"6.1","imdbVotes":"271,245","imdbID":"tt0413267","Type":"movie","DVD":"13 Nov 2007","BoxOffice":"$320,706,665","Production":"Dreamworks","Website":"N/A","Response":"True"}
```

Reply:
Okay, that looks good

Reply:
Lets leave a comment here about why the strings are capitalized so we dont forget.

Reply:
OK I added a comment explaining the capitalization of this values.